drizzle_migrations/20250130163044_mature_james_howlett.sql

```diff
@@ -0,0 +1,675 @@
+CREATE TABLE "action_categories" (
+	"created_at" timestamp (3) with time zone DEFAULT now() NOT NULL,
+	"creator_id" uuid,
+	"description" text,
+	"id" uuid PRIMARY KEY NOT NULL,
```

Comment:
Why was this file created? There was a migration file there previously. This seems unnecessary.

Reply:
This is not resolved. The current file in the repo is `20250122092015_sweet_scrambler.sql`
- https://github.com/PalisadoesFoundation/talawa-api/tree/develop-postgres/drizzle_migrations